core/src/main/java/greencity/controller/EventCommentController.java

```diff
@@ -0,0 +1,101 @@
+import greencity.dto.eventcomment.EventCommentDto;
+import greencity.dto.user.UserVO;
+import greencity.service.EventCommentService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import springfox.documentation.annotations.ApiIgnore;
+
+import javax.validation.Valid;
+
+@Validated
+@AllArgsConstructor
+@RestController
+@RequestMapping("/events/comments")
+public class EventCommentController {
+    private final EventCommentService eventCommentService;
+
+    /**
+     * Method for creating {@link greencity.dto.eventcomment.EventCommentVO}.
+     *
+     * @param eventId id of {@link greencity.dto.event.EventVO} to add comment to.
+     * @param request - dto for {@link greencity.dto.eventcomment.EventCommentVO}
+     *                entity.
+     * @return dto {@link greencity.dto.eventcomment.AddEventCommentDtoResponse}
+     */
+    @ApiOperation(value = "Add comment.")
+    @ResponseStatus(value = HttpStatus.CREATED)
+    @ApiResponses(value = {
+        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = AddEventCommentDtoRequest.class),
+        @ApiResponse(code = 303, message = HttpStatuses.SEE_OTHER),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND),
+    })
+    @PostMapping("{eventId}")
+    public ResponseEntity<AddEventCommentDtoResponse> save(@PathVariable Long eventId,
+        @Valid @RequestBody AddEventCommentDtoRequest request,
+        @ApiIgnore @CurrentUser UserVO user) {
+        return ResponseEntity
+            .status(HttpStatus.CREATED)
+            .body(eventCommentService.save(eventId, request, user));
+    }
+
+    /**
+     * Method to count active comments to certain
+     * {@link greencity.dto.event.EventVO}.
+     *
+     * @param eventId to specify {@link greencity.dto.event.EventVO}
+     * @return amount of comments
+     */
+    @ApiOperation(value = "Count comments.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND),
+    })
+    @GetMapping("/count/comments/{eventId}")
```

Comment:
I think you don't need to duplicate word "comments"
Now this controller looks like "events/comments/count/comments/id"